app-emulation/docker-credential-helpers/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>rkitover@gmail.com</email>
+		<name>Rafael Kitover</name>
+	</maintainer>
+	<longdescription>A suite of programs to use native stores to keep Docker credentials safe. Currently provides docker-credential-secretservice to use the D-Bus secret service APIs, and docker-credential-pass to use the pass framework.</longdescription>
+	<use>
+		<flag name="hardened">Build for a hardened profile.</flag>
+		<flag name="libsecret">Build secretservice helper which works with gnome-keyring or kwallet using D-Bus.</flag>
```

Comment:
We're using global `gnome-keyring` flag for this right now. It should be renamed at some point but please don't introduce temporary flags in the meantime.